deepinv/optim/optim_iterators/primal_dual_CP.py

```diff
@@ -107,8 +107,8 @@ def forward(self, x, w, cur_data_fidelity, y, physics, cur_params):
             return cur_data_fidelity.prox(p, y, physics, gamma=cur_params["stepsize"])
         else:
             p = x + cur_params["stepsize_dual"] * w
-            return cur_data_fidelity.prox_d_conjugate(
-                p, y, gamma=cur_params["stepsize_dual"]
+            return cur_data_fidelity.prox_conjugate(
```

Comment:
hmm is this correct? I think this will be prox of the conjugate of d(Ax, y) and not d(u, v)?

Reply:
I think that in order to be general and to write a function to minimize f + g, it should remains like this. If we put on the distance, it means that we assume that we took K = A (and that A is linear).

Reply:
Moreover, we could want to minimize f + g with primal dual and $K = Id$